organize/models.py

```diff
@@ -92,6 +93,20 @@ def create_event(self):
 
         return event
 
+    def has_past_team_members(self):
+        """ For repeated events, check whether there are any common
+        team members who applied to organize again
+        """
+        previous_events = Event.objects.filter(city=self.city,
+                                               country=self.get_country_display())
+        organizers = []
+        for event in previous_events:
+            for organizer in event.team.all():
+                organizers.append(organizer.email)
+
+        applicants = self.get_all_recipients()
```

Comment:
If we are using it here, let's maybe rename to something like `get_organizers_emails`?